CHANGELOG.md

```diff
@@ -23,6 +23,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
+- **CUMULUS-2557**
+  - Updated `@cumulus/aws-client/S3/moveObject` such that moveGranules operations with zero byte
+    files (0 byte files) will succeed and not throw MalformedXML errors. Updated moveGranules tests
+    in cover such operations, and added comments for the issue. 
```

Comment:
remove `Updated moveGranules tests in cover such operations, and added comments for the issue. `

Reply:
`@cumulus/aws-client/S3/moveObject` could be used for other things, so maybe:
Updated `@cumulus/aws-client/S3/moveObject` to handle zero byte files.